GettingStartedGuide.md

```diff
@@ -84,16 +122,31 @@ The demo will connect to the AWS IoT device Endpoint that you configure here.
 
 The key and certificates which will be used to establish a secure TLS connection will be stored in a special flash partition. Run the following command to create and flash the certificate partition.
 
+The following values will be needed:
+
+- `PORT`: The serial port to which the ESP32-C3 board is connected
+
+- `CA_CERT_FILEPATH`: The file path to the PEM-encoded root CA certificate,
+
+- `DEVICE_CERT_FILEPATH`: The file path to the PEM-encoded device certificate
+
+- `PRIVATE_KEY_FILEPATH`: The file path to the PEM-encoded private key.
+
+- `KEY_ALG_INFO`: The type of key algorithm being used. In the
+format Algorithm Size,
+
+  - **NOTE:** If using the
+    [AWS IoT Generated Credentials](https://docs.aws.amazon.com/iot/latest/developerguide/device-certs-create.html)
+    this value will be `rsa 2048`
+
+**NOTE** The list of chip types can be found by running `idf.py list-targets`
+The directly supported chips are the `esp32c3`, `esp32s3`, and `esp32c2`
+
 ```sh
-python managed_components/espressif__esp_secure_cert_mgr/tools/configure_esp_secure_cert.py -p PORT --keep_ds_data_on_host --ca-cert CA_CERT_FILEPATH --device-cert DEVICE_CERT_FILEPATH --private-key PRIVATE_KEY_FILEPATH --target_chip esp32c3 --secure_cert_type cust_flash
+idf.py set-target CHIP_TYPE;
```

Comment:
set-target should be done first, otherwise, endpoint and device name have to be set again.

Reply:
This line should be moved to 2.2, before step 1, before running idf.py menuconfig

Reply:
Oops, thought I had it before then, thanks for noticing!